hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1930,4 +1930,15 @@ possible configurations would overwhelm and obscure the important.
       this feature.
     </description>
   </property>
+  <property>
+    <name>hbase.regionserver.slowlog.ringbuffer.size</name>
+    <value>256</value>
```

Comment:
by having 256 per RS?
by rough estimation, how much memory it will cost? thanks
If not much, should this be a bigger value. (depends one the size estimation above)

Reply:
Rough estimation could be 400-800 KB (UTF-32) in total including all 256 entries, ideally we can increase the ringbuffer size right? May be keep it to 1024 should be good?